tests/PermissionedControllerEmissionTestMATICXPolygon.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Polygon, AaveV3PolygonAssets} from 'aave-address-book/AaveV3Polygon.sol';
+import {IEmissionManager, ITransferStrategyBase, RewardsDataTypes, IEACAggregatorProxy} from '../src/interfaces/IEmissionManager.sol';
+import {LMSetupBaseTest} from './utils/LMSetupBaseTest.sol';
+import {IOwnable} from 'solidity-utils/contracts/transparent-proxy/interfaces/IOwnable.sol';
+import {TransparentProxyFactory} from 'solidity-utils/contracts/transparent-proxy/TransparentProxyFactory.sol';
+import {Executor} from 'aave-governance-v3/contracts/payloads/Executor.sol';
+import {PermissionedPayloadsController, IPayloadsControllerCore, PayloadsControllerUtils, IPermissionedPayloadsController} from 'aave-governance-v3/contracts/payloads/PermissionedPayloadsController.sol';
+import {IERC20} from 'forge-std/interfaces/IERC20.sol';
+
```

Comment:
@Cycxyz I would put a comment on the generator which has the makefile command to run the tests, both for setup and update